installer/alpine/build.sh

```diff
@@ -14,6 +14,7 @@ build_initramfs() {
 
 	cat >/etc/mkinitfs/features.d/network.modules <<-EOF
 		kernel/drivers/net/ethernet
+		kernel/drivers/net/phy
```

Comment:
hmm `/phy` drivers should already be pulled as the alpine scripts are able to detect module dependencies. Maybe we don't have the correct realtek driver?

```
[~/go/src/github.com/tinkerbell/osie/installer/alpine/assets-x86_64]─[manny@dellnix]> 
mkdir chroot
                                                                                                                                                            
[~/go/src/github.com/tinkerbell/osie/installer/alpine/assets-x86_64]─[manny@dellnix]> 
tar -xf initramfs -C chroot/
no candidates

[~/go/src/github.com/tinkerbell/osie/installer/alpine/assets-x86_64]─[manny@dellnix]> 
ls chroot/lib/modules/5.4.52-1-lts/kernel/drivers/net/phy/
fixed_phy.ko  libphy.ko  phylink.ko
                                                                                                                                                            
[~/go/src/github.com/tinkerbell/osie/installer/alpine/assets-x86_64]─[manny@dellnix]> 
```

Reply:
I'd rather make sure we have the correct realtek driver or debug why the realtek's phy isn't being added.

Reply:
There are a number of examples of underlying `phy` drivers missing.. `lib/modules/5.4.52-1-lts/kernel/drivers/net/phy/broadcom.ko` being another.. we include all of the other `net/ethernet` drivers but not the underlying physical driver.

Reply:
ah very true! thanks for the reminder. lgtm, all we need is your DCO @thebsdbox